src/transformers/modeling_tf_utils.py

```diff
@@ -884,6 +885,93 @@ def load_repo_checkpoint(self, repo_path_or_name):
         # set it directly, but the user can pass it to fit().
         return {"epoch": extra_data["epoch"]}
 
+    def prepare_tf_dataset(
+        self,
+        dataset: Any,
+        batch_size: int,
+        shuffle: bool,
+        tokenizer: Optional[PreTrainedTokenizer] = None,
+        collate_fn: Optional[Callable] = None,
+        collate_fn_args: Optional[dict] = None,
+        drop_remainder: Optional[bool] = None,
+        prefetch: bool = True,
+    ):
+        """
+        Wraps a HuggingFace `Dataset` as a `tf.data.Dataset` with collation and batching. This method is designed to
```

Comment:
```suggestion
        Wraps a HuggingFace `datasets.Dataset` as a `tf.data.Dataset` with collation and batching. This method is designed to
```
and also add a link to the doc